pages/common/samtools.md

```diff
@@ -0,0 +1,28 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM file to BAM (use - for stdin):
+
+`samtools view -bS {{input.sam}} > {{output.bam}}`
+
+- Print a region of a file, include the file header:
+
+`samtools view -h {{input}} chr:start-end > {{output.sam}}`
```

Comment:
Chromosome. I would hope that this is pretty common knowledge for anyone using the tool, but I could extend it to `chromosome:start-end`.